python-frontend/src/main/java/org/sonar/python/semantic/FunctionSymbolImpl.java

```diff
@@ -122,6 +128,21 @@ public FunctionSymbolImpl(SymbolsProtos.FunctionSymbol functionSymbolProto, bool
     this.validForPythonVersions = new HashSet<>(validFor);
   }
 
+  public FunctionSymbolImpl(FunctionDescriptor functionDescriptor, ProjectLevelSymbolTable projectLevelSymbolTable, String symbolName) {
+    super(symbolName, functionDescriptor.fullyQualifiedName());
+    setKind(Kind.FUNCTION);
+    isInstanceMethod = functionDescriptor.isInstanceMethod();
+    isAsynchronous = functionDescriptor.isAsynchronous();
+    hasDecorators = functionDescriptor.hasDecorators();
+    decorators = functionDescriptor.decorators();
+    annotatedReturnTypeName = functionDescriptor.annotatedReturnTypeName();
+    functionDefinitionLocation = functionDescriptor.definitionLocation();
+    parameters.addAll(functionDescriptor.parameters().stream().map(p -> DescriptorUtils.functionParameter(p, projectLevelSymbolTable)).collect(Collectors.toList()));
+    hasVariadicParameter = parameters.stream().anyMatch(FunctionSymbol.Parameter::isVariadic);
+    // TODO: Will no longer be true once SONARPY-647 is fixed
```

Comment:
I assume once we solve SONARPY-647, it might be possible to have stubs in the project symbol table if a module exports some stdlib functions (or any kind of stubbed function really).